cmd/infracost/breakdown_test.go

```diff
@@ -262,6 +262,8 @@ func TestBreakdownInitFlagsError(t *testing.T) {
 			"breakdown",
 			"--path",
 			path.Join("./testdata", testutil.CalcGoldenFileTestdataDirName()),
+			"--project-type",
+			"terraform_cli",
```

Comment:
do we want to force `terraform_cli` here or should there be `hcl` coverage?

Reply:
Done, and added a separate golden file with the `terraform_dir` error that shows here.